src/rpcblockchain.cpp

```diff
@@ -397,21 +397,21 @@ UniValue gettxout(const UniValue& params, bool fHelp)
             "3. includemempool  (boolean, optional) Whether to included the mem pool\n"
             "\nResult:\n"
             "{\n"
-            "  \"bestblock\" : \"hash\",    (string) the block hash\n"
-            "  \"confirmations\" : n,       (numeric) The number of confirmations\n"
-            "  \"value\" : x.xxx,           (numeric) The transaction value in btc\n"
-            "  \"scriptPubKey\" : {         (json object)\n"
-            "     \"asm\" : \"code\",       (string) \n"
-            "     \"hex\" : \"hex\",        (string) \n"
-            "     \"reqSigs\" : n,          (numeric) Number of required signatures\n"
+            "  \"bestblock\" : \"hash\",     (string) the block hash\n"
```

Comment:
If a line has four escaped quotes, the description should appear two characters further to the right than if it has only two escaped quotes (i.e. the escapes are nonprinting characters).